FinalXmL/FinalXmL/Pages/Index.cshtml

```diff
@@ -2,52 +2,48 @@
 @model IndexModel
 @{
     ViewData["Title"] = "Home page";
-    var welcomes = (QuickType.Welcome[])ViewData["Welcome"];
-    var geodatas = (QuickType1.Geonamedata[])ViewData["Geoname"];
-}
-
-<div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <table border="1">
-        <tr>
-            <th>Name</th>
-            <th>Code</th>
 
+    //var countryList = (QuickTypeCList.CountryList[])ViewData["CountryList"];
+    //var cityList = (QuickTypeCityList.CityList[])ViewData["CityList"];
```

Comment:
Remove commented-out code.  You're in your own branch; you can do what you want without impacting master.